osd_cluster_admin/osd_hibernate_resume/osd-overview-hibernating-resuming-cluster.adoc

```diff
@@ -0,0 +1,43 @@
+
+toc::[]
+
+As a cluster administrator, you might need to hibernate a {product-title} cluster for a period and resume it later to conserve resources.
+
+[IMPORTANT]
+====
+Hibernating a ready cluster and resuming a hibernating cluster are Technology Preview features only. The hibernate and resume features for {product-title} clusters are provided as Technology Preview features as explained in link:https://access.redhat.com/articles/7012966[Technical Preview: Hibernating and Resuming Clusters]. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. 
+
+During the entire period of hibernation and till the resumed cluster is ready, the cluster is in link:https://docs.openshift.com/rosa/rosa_architecture/rosa_policy_service_definition/rosa-service-definition.html#rosa-limited-support_rosa-service-definition[Limited Support] status under {product-title} SLA.  For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope]
+====
+
+A hibernating cluster requires significantly fewer resources than one that is running, potentially leading to lowered provider costs through the strategic movement of clusters in and out of a hibernating state. Hibernating a ready cluster on-demand can notably reduce infrastructure expenses, including those related to EC2 instances and network devices. 
+
+This approach is particularly effective for development or test clusters that are only intermittently used by end users. However, it is important to note that hibernation does not lead to cost savings in all aspects. While it helps curtail expenses for items such as EC2 instances and network devices, expenses tied to EBS and S3 continue to accrue during the hibernation period. This is particularly relevant when you have an annual commitment or a prepaid quota in place, as these costs are not mitigated by hibernation. 
+
+Therefore, careful consideration of your specific cost structure and usage patterns is necessary when determining the most effective.
+
+[NOTE]
+====
+You must resume the cluster prior to the 60 days of hibernation. Once you hibernate your cluster, the system sends up to 3 reminders within the 60 days limit or as OpenShift Container Platform (OCP) End-of-Life (EOL) is approached.
+====
+
+During hibernation:
+
+* Applications are not accessible
+* Cluster admin functions except resume are unavailable
+* Cluster updates do not happen. You should monitor EOL dates and ensure cluster is updated
+
+[NOTE]
+====
+If you are unable to successfully resume a hibernating cluster, follow DR strategy.
```

Comment:
Could you please help with the link for **DR strategy**? So that we can hyperlink, DR strategy. Thank you!